tests/test_bt_merge.cpp

```diff
@@ -1,12 +1,76 @@
+#include <rapidcheck.h>
+
 #include <catch2/catch_test_macros.hpp>
 
 #include "catch2_bt_format.hpp"
+#include "rapidcheck/catch.h"
 #include "session/bt_merge.hpp"
 
 using oxenc::bt_dict;
 using oxenc::bt_list;
 using oxenc::bt_value;
 
+namespace rc {
+
+template <>
+class Arbitrary<std::string_view> {
+  public:
+    static rc::Gen<std::string_view> arbitrary() {
+
+        return rc::gen::map(rc::gen::arbitrary<std::string>(), [](const std::string& s) {
+            return std::string_view(s);
+        });
+    }
+};
+
+Gen<bt_value> genBtValue() {
+    return gen::oneOf<bt_value>(
+            gen::map(
+                    gen::resize(20, gen::arbitrary<std::string>()),
```

Comment:
is there a way to give a random size instead of a hard coded `20`?

Reply:
is there a convenient way of printing a `bt_value`? I have a custom `print_bt_value()` function in https://github.com/oxen-io/libsession-util/issues/13 but I don't know if something similar is already implemented in libsessioin-util?

Reply:
`bt_serialize` will give a std::string representation of it, but it will contain non printable characters if bytestrings have those in them. you could combine `bt_serialize` with a function to replace non printable chars with a placeholder char.

Reply:
Thanks!